test/system/comment_test.rb

```diff
@@ -20,12 +20,13 @@ def get_path(page_type, path)
     page_type == :wiki ? path + '/comments' : path
   end
 
-  # page_types are wiki, research note, question:
-  page_types.each do |page_type, node_name|
-    page_type_string = page_type.to_s
-    comment_text = 'woot woot'
-    comment_response_text = 'wooly woot'
+  # weird syntax, i know. 
+  #   most comment tests we can simply test on Research Note pages.
+  #   that's what this block is for.
 
+  #   other comment tests can ALSO be tested on Wikis and Questions.
+  #   scroll past this block for those tests.
+  { :note => :comment_note }.each do |page_type, node_name|
     test "#{page_type_string}: addComment(comment_text)" do
```

Comment:
This block just runs tests on research notes (most tests will now live in this block).